lib/health-data-standards/models/patient_history.rb

```diff
@@ -0,0 +1,4 @@
+class PatientHistory < Entry
```

Comment:
What is the purpose of this model

Reply:
This was in order to get namespacing identical to FamilyHistory and Visit::FamilyHistory.  The patern was matched from FamilyHistory in Health-Data-Standards

Reply:
This isn't adding any namespacing so I don't understand why it's necessary.

Reply:
Will agree that it seems suspicious